src/legacy/ui/public/chrome/api/badge.test.ts

```diff
@@ -21,11 +21,12 @@ import * as Rx from 'rxjs';
 
 import { ChromeBadge } from 'src/core/public/chrome';
 import { chromeServiceMock } from '../../../../../core/public/mocks';
-import { __newPlatformSetup__, initChromeBadgeApi } from './badge';
+import { initChromeBadgeApi } from './badge';
+import { __newPlatformSetup__ } from 'ui/new_platform';
```

Comment:
Can you explain why this is `__newPlatformSetup__` instead of `onSetup`?

Reply:
As I understand, `onSetup` is when you want to consume `core` and `plugins`. And `__newPlatformSetup__` is what platform calls to provide `core` and `plugins`.

Reply:
@eliperelman These are only in tests where we need to supply the test with our mocked version of CoreSetup. This seemed simpler than mocking out onSetup itself to call callbacks with a mock. If you feel strongly otherwise, I'd be fine with changing this to a mock of onSetup.

Same for all below, these are only in tests.

Reply:
This PR has been updated to change to be able to import npStart and npSetup directly.